packages/shared/sierra-client/tests/http-sierra-client.test.ts

```diff
@@ -7,6 +7,7 @@ describe('HTTP sierra client', () => {
   let client: HttpSierraClient;
 
   beforeEach(() => {
+    // @ts-ignore
```

Comment:
What are we ignoring here? Can we be more specific in the annotations?

Reply:
I'm not sure - it should've been failing before but wasn't because the type dependencies weren't installed correctly, and I just copied it from the Auth0 Client test. I am planning to get rid of Moxios in my next PR so not too bothered about it...